ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/result-sort/presentation.tsx

```diff
@@ -29,7 +29,6 @@ type Props = {
 const render = ({ removeSort, saveSort, hasSort, collection }: Props) => {
   return (
     <div className="min-w-120">
-      {/* @ts-expect-error ts-migrate(2322) FIXME: Property 'collection' does not exist on type 'Intr... Remove this comment to see the full error message */}
       <SortSelections collection={collection} />
```

Comment:
📓 Added the ts-nocheck because putting the error above this line doesn't work. Also, this component doesn't have a `collection` property on it. Wasn't sure how we'd want to fix this.